app/javascript/components/patient/Patient.js

```diff
@@ -151,12 +160,11 @@ class Patient extends React.Component {
             />
           )}
           <Col sm={12}>
-            <h3>
-              <span aria-label={formatName(this.props.details)} className="pr-2">
-                {formatName(this.props.details)}
-              </span>
+            <Heading level={rootHeaderLevel + 1} className="secondary-title">
```

Comment:
As indicated in other comment, I think we should treat the prop passed in as the root level for this component, meaning here we should use `rootHeaderLevel`, and all the other Headings here should be bumped down by one.

Reply:
fixed.